src/widgets/window.js

```diff
@@ -96,47 +95,50 @@ var FlatsealWindow = GObject.registerClass({
         const allPermissions = this._permissions.getAll();
 
         this._detailsHeaderButton = new FlatsealDetailsButton(this._permissions);
-        this._startHeaderBox.add(this._detailsHeaderButton);
+        this._startHeaderBox.append(this._detailsHeaderButton);
         this._resetHeaderButton = new FlatsealResetButton(this._permissions);
-        this._endHeaderBox.add(this._resetHeaderButton);
+        this._endHeaderBox.append(this._resetHeaderButton);
 
         this._detailsActionButton = new FlatsealDetailsButton(this._permissions);
-        this._startActionBox.add(this._detailsActionButton);
+        this._startActionBox.append(this._detailsActionButton);
         const resetActionButton = new FlatsealResetButton(this._permissions);
-        this._endActionBox.add(resetActionButton);
+        this._endActionBox.append(resetActionButton);
 
-        this._undoPopup = new FlatsealUndoPopup(this._permissions);
-        this._undoPopupBox.add(this._undoPopup);
+        this._toast = new Adw.Toast();
```

Comment:
can't we sub class Adw.Toast and keep these things there?

Reply:
Yes, but I feel it is overkill if we only need to set three lines of properties